package.json

```diff
@@ -100,7 +92,20 @@
     "lint": "node ./node_modules/eslint/bin/eslint.js ./static/js",
     "scss_lint": "node ./node_modules/sass-lint/bin/sass-lint.js --verbose --no-exit",
     "test": "./js_test.sh",
-    "coverage": "node ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha -- --compilers js:babel-register static/js/global_init.js 'static/**/*/*_test.js'",
+    "coverage": "node ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha --report lcovonly -- --compilers js:babel-register static/js/global_init.js 'static/**/*/*_test.js' && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
     "flow": "flow check"
+  },
+  "devDependencies": {
+    "eslint": "^2.13.1",
```

Comment:
Can you keep these in `dependencies`? We've decided to keep everything there for simplicity's sake, and since webpack will already exclude the packages we don't use